packages/mdc-image-list/_mixins.scss

```diff
@@ -26,13 +27,9 @@
   }
 }
 
-@mixin mdc-image-list-corner-radius($radius) {
+@mixin mdc-image-list-shape-radius($radius, $rtl-reflexive: false) {
   .mdc-image-list__image {
-    border-radius: $radius;
-  }
-
-  .mdc-image-list__supporting {
-    border-radius: 0 0 $radius $radius;
+    @include mdc-shape-radius($radius, $rtl-reflexive);
```

Comment:
This lost the details of how `.mdc-image-list__supporting` was styled, which constrains to only the bottom corners (the top corners are always flush with the image; only the bottom is considered to be along the boundary of the item).

Reply:
With the previous implementation __supporting border-radius style was doing nothing since it doesn't have any background. Also, the corner-radius mixin applies border radius to 4 corners of image with and without existence of __supporting element.

Reply:
It does have a background when the image list uses overlaid labels with text protection. See the first example under Demos at https://material-components.github.io/material-components-web-catalog/#/component/image-list

Reply:
Good catch. This wasn't applying with previous implementation - corrected that.